lib/ui/widgets/custom_text_form_field.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+
+class CustomTextFormField extends StatefulWidget {
+  final bool includeTextField;
+  final String labelText;
+  final String hintText;
```

Comment:
**nitpick:** Labels and texts are always texts ;) i think we should remove the suffix

Reply:
true